doit/tools.py

```diff
@@ -49,15 +49,16 @@ class config_changed(object):
     """check if passed config was modified
     @var config (str) or (dict)
     """
-    def __init__(self, config):
+    def __init__(self, config, encoder=None):
```

Comment:
here too need to document parameter. just mention it is the same as in `json.dumps()`